motion/p5js/motion-triangle/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Daniele Maselli
+author-url: https://github.com/DanieleMaselli
+title: motion-triangle
+tags: [P5.js, p5.Vector, color, triangle]
+permalink: /color/motion/
```

Comment:
the permalink can not be `/color/motion/` This is to broad. Please give it a specific name. Be creative. If you commit this into the folder motion the permalink should also start with motion.